src/app/topic/topic.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { HtmlTagDefinition, ArrayType } from '@angular/compiler';
+
+export interface Topic{
+  id: number,
+  title: string,
+  content: string,
+  videolink: string,
+  files: object,
```

Comment:
This `files` should be an array of `File` interface, so `Array<File>`. 
Define `File` after that